src/Illuminate/Container/CircularReferenceException.php

```diff
@@ -0,0 +1,36 @@
+<?php namespace Illuminate\Container;
+
+use Exception;
+
+class CircularReferenceException extends Exception {
+
+	/**
+	 * @var array
+	 */
+	protected $buildStack;
+
+    /**
+     *  Create a new Circular Reference Exception instance
```

Comment:
And a full stop, and all lower case apart from the first letter in this sentence.